docs/admin/extensible-admission-controllers.md

```diff
@@ -176,8 +176,12 @@ user._ If there is an error encountered when calling an external admission
 webhook, that request is ignored and will not be used to approve/deny the
 admission request.
 
-**Note:** The admission chain depends solely on the order of the
-`--admission-control` option passed to `kube-apiserver`.
+**Note** Before v1.10, the admission chain depends solely on the order of the
+`--admission-control` option passed to `kube-apiserver`. Starting from v1.10,
+the `--admission-control` option is replaced by the `--enable-admission-plugins`
+and the `--disable-admission-plugins` options where the order of plugins for
+these two options no longer matters.
+{. note}
```

Comment:
Let's rewrite the references to versions a bit more clearly and consistently (and so adding a couple more copyedits for readability):
s/Before v1.10/In Kubernetes versions earlier than 1.10, ...
s/solely/only
s/Starting from v1.10/In versions 1.10 and later
l182: s/options where the/options. The